packages/rich-text/src/RichTextEditor.mdx

```diff
@@ -10,10 +10,109 @@ import Readme from '../README.md';
 
 import { Playground } from 'docz';
 import { RichTextEditor } from './index';
+import {
+  createFakeFieldAPI,
+  ActionsPlayground,
+  createFakeSpaceAPI,
+  createFakeLocalesAPI,
+  createFakeNavigatorAPI,
+} from '@contentful/field-editor-test-utils';
+import { BLOCKS } from '@contentful/rich-text-types';
+import { document, block, text } from './helpers/nodeFactory'
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
 import { css } from 'emotion';
 
 ## In Action
 
 <Playground>
-  <RichTextEditor />
+  {() => {
+    const rtPreviewStyle = css({
+      backgroundColor: 'whitesmoke',
+      padding: '0 0.5em'
+    });
+    const newEntitySelectorDummyDialog = (fnName, type) => async () => {
+      return confirm(`sdk.dialogs.${fnName}()\nSimulate selecting a random entity or cancel?`)
```

Comment:
Most of this is just copied verbatim from the file in `master`. I think it's okay to have some unused mocks in here since we know they'll be used rather soon as we develop forthcoming plugins